### ISSUE TITLE
AndroidManifest.xml merge conflict when added to cordova plugin

### ISSUE THREAD

**Issue by nacho4d** [watson-developer-cloud/speech-android-sdk#22]
I am not an expert in Android but somehow I have managed to add speech-android-sdk to a cordova plugin(after some changes).

When attempting to build for the android platform it fails in step `:mergeDebugAssets` saying that android:theme is present in two manifests (app's AndroidManifest.xml and speech-android-wrapper's AndroidManifest.xml) and cannot be merged. The CLI suggests to use 'tools:replace=...' to solve the conflict

However, I think, this aar does **not** need `<application>` tag since is meant to be added as a library. The one who should have `<application>` tag in its AndroidManifest.xml is the host.

I think below lines should be removed to avoid extra settings in the host application. 

https://github.com/watson-developer-cloud/speech-android-sdk/blob/master/speech-android-wrapper/src/main/AndroidManifest.xml#L10-L15

```
<application
    android:allowBackup="true"
    android:icon="@mipmap/ic_launcher"
    android:label="@string/app_name"
    android:theme="@style/AppTheme" android:name="com.ibm.watson.developer_cloud.android.speech_to_text.v1.WatsonSDK">
</application>
```

**Comment by mihui** [comment]
I also had that: https://github.com/watson-developer-cloud/speech-android-sdk/commit/4957ce0e336f7c831af0f2e69eabfc99a232517d

**Comment by nacho4d** [comment]
I am closing this issue since it was addressed already in 72eee987dbaa95203d747896bf5349be5bfd15dd
